deltacat/tests/utils/test_daft.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from deltacat.types.media import ContentEncoding, ContentType
+from deltacat.utils.daft import daft_s3_file_to_table
+
+
+class TestDaftParquetReader(unittest.TestCase):
+    URL = "s3://daft-public-data/test_fixtures/parquet-dev/mvp.parquet"  # Maybe change this to something deltacat hosts
```

Comment:
This test will start failing when you remove this S3 file from that location. Can you use a local file instead?

Reply:
fixed!